### DISCUSSION TITLE
Option to set the behaviour in case of empty input

### DISCUSSION THREAD

**Discussion by justice47** [dm4t2/vue-currency-input#369] [Q&A]
Currently, even if `valueRange` has been set up, in case of empty input `null` is emitted.
If there is any way to emit the `min` value set up in the `valueRange` in such a case?
Thanks!

**Comment by dm4t2** [comment]
Hi, you can set a min value with `setValue` inside a watcher on `props.modelValue`:

https://dm4t2.github.io/vue-currency-input/guide.html#external-props-changes
